jax/_src/numpy/lax_numpy.py

```diff
@@ -7443,9 +7466,33 @@ def cross(a, b, axisa: int = -1, axisb: int = -1, axisc: int = -1,
   return moveaxis(c, 0, axisc)
 
 
-@util.implements(np.kron)
 @jit
 def kron(a: ArrayLike, b: ArrayLike) -> Array:
+  """Compute the Kronecker product of two input arrays.
+
+  JAX implementation of :func:`numpy.kron`.
+
+  Args:
+    a: first input array.
+    b: second input array.
+
+  Returns:
+    A new array representing the Kronecker product of the inputs `a` and `b`.
```

Comment:
Mention the shape of the output is the elementwise product of the input shapes.

Reply:
Done